src/components/OrderConfirmation/OrderConfirmation.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { OrderItem } from '..';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+export class OrderConfirmation extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClose = this.handleClose.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.addToOrder = this.addToOrder.bind(this);
+  }
+
+  handleClose() {
+    const { closeModal } = this.props;
+    closeModal();
+  }
+
+  addToOrder() {
+    const { addToOrder } = this.props;
+    addToOrder();
+    this.handleClose();
+  }
+
+  handleChange(e) {
+    const { updateOrderOwner } = this.props;
+    updateOrderOwner(e.target.value);
+  }
+
+  render() {
+    const { open, sandwich } = this.props;
+    const { name, bread, total, isHot } = sandwich;
```

Comment:
You could destructure on the line above for `sandwich` and remove this line 👍 

```js
const { open, sandwich: { name, bread, total, isHot } } = this.props;
```

Reply:
this is great 👍